### PR TITLE
Watch single namespace

### PR DESCRIPTION
**Pull request by eye0fra**
Add the ability to run the eventrounter as a namespaced application.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -105,9 +106,8 @@ func loadConfig() kubernetes.Interface {
 // main entry point of the program
 func main() {
 	var wg sync.WaitGroup
-
 	clientset := loadConfig()
-	sharedInformers := informers.NewSharedInformerFactory(clientset, viper.GetDuration("resync-interval"))
+	sharedInformers := informers.NewSharedInformerFactoryWithOptions(clientset, viper.GetDuration("resync-interval"), informers.WithNamespace(viper.GetString("WATCH_NAMESPACE")))
```
Comment (jcantrill) [inline main.go:110]: I'm unfamiliar with the behavior but what happens if the var is missing?  The viper docs do not seem to be specific

**Review by jcantrill — COMMENTED** [review]
/ok-to-test

File: main.go (hunk 1 above)
Comment (eye0fra) [reply main.go:110]: Hi @jcantrill, it works like it was not defined, so it keeps the previous behaviour.

**Pull request merged**